%YAML:1.0
---
camera_matrix_left: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 7.1328802286613575e+02, 0., 3.1665418121053369e+02, 0.,
       7.1321046937525045e+02, 2.4365128127568744e+02, 0., 0., 1. ]
camera_matrix_right: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 7.1328802286613575e+02, 0., 3.1492535322241656e+02, 0.,
       7.1321046937525045e+02, 2.3915313459128370e+02, 0., 0., 1. ]
dist_coefficient_left: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 2.8340134910549403e-02, -3.4265527232619275e-02, 0., 0.,
       -1.6636818346163190e-01 ]
dist_coefficient_right: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 2.3860314543105440e-02, -2.6961412410807573e-02, 0., 0.,
       -4.9835863444908618e-01 ]
rotation: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 9.9742319602743590e-01, 2.5602848903990283e-02,
       -6.7018371767855675e-02, -2.5798904816819124e-02,
       9.9966502741228125e-01, -2.0614264613581794e-03,
       6.6943144060223073e-02, 3.7851151636805118e-03,
       9.9774961206032542e-01 ]
translation: !!opencv-matrix
   rows: 3
   cols: 1
   dt: d
   data: [ -3.5948163751966547e+01, 8.9931697506196140e-01,
       -1.9968479145207745e+00 ]
essential: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 8.6866165369916953e-03, 1.9995830435268289e+00,
       8.9317680785703635e-01, 4.1478067586635464e-01,
       8.4942944284315203e-02, 3.6001091933704892e+01,
       3.0423643468385018e-02, -3.5959147179160112e+01,
       1.3437525536738668e-01 ]
fundamental: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 5.4369612872139234e-08, 1.2516769584880941e-05,
       9.2062337968625192e-04, 2.5963983665931154e-06,
       5.3177430014175381e-07, 1.5979177099622960e-01,
       -5.0223354228704897e-04, -1.6462524073300366e-01, 1. ]
rectification_left: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 9.9993329184879931e-01, 8.0294674551300129e-04,
       -1.1522461930842711e-02, -8.2050085226941492e-04,
       9.9999850997787965e-01, -1.5188220342965554e-03,
       1.1521225228910092e-02, 1.5281749063211761e-03,
       9.9993246074456466e-01 ]
rectification_right: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 9.9814943413163004e-01, -2.4970753331286052e-02,
       5.5445185731266854e-02, 2.4886213152507348e-02,
       9.9968783685259899e-01, 2.2147785215118261e-03,
       -5.5483182475724861e-02, -8.3085921758521860e-04,
       9.9845927615267616e-01 ]
projection_left: !!opencv-matrix
   rows: 3
   cols: 4
   dt: d
   data: [ 7.1321046937525045e+02, 0., 2.9662372970581055e+02, 0., 0.,
       7.1321046937525045e+02, 2.4102666091918945e+02, 0., 0., 0., 1.,
       0. ]
projection_right: !!opencv-matrix
   rows: 3
   cols: 4
   dt: d
   data: [ 7.1321046937525045e+02, 0., 2.9662372970581055e+02,
       -2.5686140637873021e+04, 0., 7.1321046937525045e+02,
       2.4102666091918945e+02, 0., 0., 0., 1., 0. ]
disparity: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [ 1., 0., 0., -2.9662372970581055e+02, 0., 1., 0.,
       -2.4102666091918945e+02, 0., 0., 0., 7.1321046937525045e+02, 0.,
       0., 2.7766353825987185e-02, 0. ]
